processing/src/main/java/org/apache/druid/math/expr/BuiltInExprMacros.java

```diff
@@ -144,4 +145,47 @@ public Object getLiteralValue()
       }
     }
   }
+
+  public static class StringDecodeBase64UTFExprMacro implements ExprMacroTable.ExprMacro
```

Comment:
It would be useful to implement `getOutputType`, `isLiteral`, `isNullLiteral`, and `getLiteralValue` similar to `ComplexDecodeBase64Expression`.

Reply:
General Q, can this be implemented as an extension of `UnivariateFunction`

Reply:
`BaseScalarUnivariateMacroFunctionExpr` is the macro version of right thing to extend here since this is a single arugment function, +1 for re-using it since it would simplify things a bit here

Reply:
Fixed